src/main/java/org/jabref/gui/keyboard/KeyBinding.java

```diff
@@ -55,7 +55,8 @@ public enum KeyBinding {
     FILE_LIST_EDITOR_MOVE_ENTRY_DOWN("File list editor, move entry down", Localization.lang("File list editor, move entry down"), "ctrl+DOWN", KeyBindingCategory.VIEW),
     FILE_LIST_EDITOR_MOVE_ENTRY_UP("File list editor, move entry up", Localization.lang("File list editor, move entry up"), "ctrl+UP", KeyBindingCategory.VIEW),
     FIND_UNLINKED_FILES("Search for unlinked local files", Localization.lang("Search for unlinked local files"), "shift+F7", KeyBindingCategory.QUALITY),
-    FOCUS_ENTRY_TABLE("Focus entry table", Localization.lang("Focus entry table"), "alt+1", KeyBindingCategory.VIEW),
+    FOCUS_ENTRY_TABLE("Focus entry table", Localization.lang("Focus entry table"), "TAB", KeyBindingCategory.VIEW),
```

Comment:
Why did you change this?

Reply:
this is changed to match the Tasks of focusing on entry table after pressing tab from the pull request #9863

Reply:
Is it still possible to navigate away from the entry table? I would see tab as cycling though different ui components: Main menu entries, groups, main table, entry editor, ...

It should be possible to focus these elements using quick shortcut. Alr+1 was such a shortcut.

Reply:
I think there is a bit of a misunderstanding of the issue:
Tab and Shift Tab are coming from the operating system and mean:
Tab: Forward navigation between controls
Shift + Tab: Backwards navigation between controls 

The guy means: If you have the entry editor open, pressing shift + tab should navigate to the group. And if your focus is on the group; Navigation with tab should go forward to the 
 But I think this might be difficult. It depends on the order of the Nodes in the javafx scencegraph. Focus model etc.
I would rather add two extra shortcuts (like you did), but with separate shortcuts: like alt+1 or alt+2

Reply:
Hi Siedlerchr, we can use alt + 1 to focus on the entry table (like before) and alt + 2 to focus on the group list

Reply:
I think the reporter @DvP17 at https://github.com/JabRef/jabref/issues/9863 really meant a proper <kbd>Tab</kdb> cycling order. I agree, however, that this is very difficult to implement. A workaround using Alt+1 and Alt+2 is OK for me.

Note that the hotkey system was revised at https://github.com/JabRef/jabref/pull/1525. Updated at https://github.com/JabRef/jabref/pull/1605.

| Keys - new | Keys - old | Function |
| --- | --- | --- |
| F7 | alt + f | Automatically set file links |
| F8 | ctrl + shift + F7 | Cleanup entries |
| alt + 1 | ctrl + shift + E | Focus entry table |
| alt + 2 | ctrl + F9 | Toggle entry/preview editor |
| alt + 3 | ctrl + shift + G | Toggle groups |
| alt + 4 | F5 | web search |
| alt + 0 | - | Open Openoffice/LibreOffice connection |
| ctrl + shift + F7 | ctrl + f4 | Synchronize file links |
| ctrl + shift R | - | Technical report |
| - | alt + P | Print entry preview |
| - | ctrl + alt + T | Hide/show toolbar |
| - | ctrl+P | Edit preamble |

Maybe, we should think about propr key binding defaults. Alt+Number is used for both toggle and focus in the implementation of 2016.

Reply:
Note that we have a blog article on a revised hotkey system in 2016: https://blog.jabref.org/2016/08/15/HotkeyRevision/